### PR TITLE
CHANGELOG entry for gRPC instrumentation

### PR DESCRIPTION
**Pull request by fallwith**
update the 8.10 notes to mention gRPC

### CODE REVIEW

**Review by tannalynn — COMMENTED** [review]
Maybe we should make a table or list of the new configuration options and explain what they each do. Might be helpful for people, especially since we have the deny list one too, which is a bit more configuration than we have typically.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
   ## v8.10.0
 
 
+  * **gRPC instrumentation**
+
+    The agent will now instrument [gRPC](https://grpc.io/) activity performed by clients and servers that use the [grpc](https://rubygems.org/gems/grpc) RubyGem. Instrumentation is automatic and enabled by default, so gRPC users should not need to modify any existing application code or agent configuration to benefit from the instrumentation. The instrumentation makes use of New Relic Distributed Tracing for a comprehensive overview of all gRPC traffic taking place across multiple monitored applications. The Ruby agent's use of Distributed Tracing is compatible with other programming language based New Relic agents that support gRPC.
```
Comment (kaylareopelle) [inline CHANGELOG.md:9]:
```suggestion
    The agent will now instrument [gRPC](https://grpc.io/) activity performed by clients and servers that use the [grpc](https://rubygems.org/gems/grpc) RubyGem. Instrumentation is automatic and enabled by default, so gRPC users should not need to modify any existing application code or agent configuration to benefit from the instrumentation. The instrumentation makes use of New Relic Distributed Tracing for a comprehensive overview of all gRPC traffic taking place across multiple monitored applications. The Ruby agent's use of Distributed Tracing is compatible with other New Relic language agents that support gRPC.
```

Comment (kaylareopelle) [reply CHANGELOG.md:9]: I don't think one exists yet, but when a docs page is added for gRPC instrumentation, could you add a link to it in the Changelog?

I think the doc could have more details about external segments being generated by the client, transactions generated by the server and the attributes each publish.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,19 @@
...
```
Comment (kaylareopelle) [inline CHANGELOG.md:9]: I'm sorry I didn't catch this last time. Distributed tracing follows a W3C standard, so labeling it as `New Relic` isn't quite appropriate. Infinite tracing, on the other hand, could probably always be prefixed with `New Relic`.
 
```suggestion
    The agent will now instrument [gRPC](https://grpc.io/) activity performed by clients and servers that use the [grpc](https://rubygems.org/gems/grpc) RubyGem. Instrumentation is automatic and enabled by default, so gRPC users should not need to modify any existing application code or agent configuration to benefit from the instrumentation. The instrumentation makes use of distributed tracing for a comprehensive overview of all gRPC traffic taking place across multiple monitored applications. This allows you to observer your client and server activity using any service that adheres to the W3C standard.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,19 @@
...
+
+    The following new configuration parameters have been added for gRPC. All are optional.
+
+    | Configuration name | Default | Behavior |
+    | ----------- | ----------- |----------- |
+    | `instrumentation.grpc_client` | auto | Set to 'disabled' to disable, set to 'chain' if there are module prepending conflicts |
+    | `instrumentation.grpc_server` | auto | Set to 'disabled' to disable, set to 'chain' if there are module prepending conflicts |
+    | `instrumentation.grpc.host_denylist` | "" |  Provide a comma delimited list of host regex patterns (ex: "private.com$,exception.*" to ignore |
```
Comment (kaylareopelle) [inline CHANGELOG.md:17]:
```suggestion
    | `instrumentation.grpc.host_denylist` | "" |  Provide a comma delimited list of host regex patterns (ex: "private.com$,exception.*") |
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,19 @@
   ## v8.10.0
 
 
+  * **gRPC instrumentation**
```
Comment (kaylareopelle) [inline CHANGELOG.md:7]:
```suggestion
  * **New gRPC instrumentation**
```
Take or leave this one!

**Review by kaylareopelle — CHANGES_REQUESTED** [review]

**Review by kaylareopelle — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (fallwith) [reply CHANGELOG.md:7]: It reads the same both ways to me. If I see it without the "new", I might think that we've made updates to existing instrumentation, which is not desired. If I see it with the "new", I might think there have been new enhancements added to the existing instrumentation, which is not desired. I'll commit the suggestion to decisively go with one.

**Review by kaylareopelle — APPROVED** [review]

**Pull request merged**